src/TuxOwnerClass.js

```diff
@@ -0,0 +1,37 @@
+var React = require('react');
+var getOwnerPropsMixin = require('./TuxGetOwnerPropsMixin');
+var StoreMixinGenerator = require('./TuxStoreMixinGenerator');
+var assign = require('object-assign');
+
+//createOwnerClass FUNCTION creates an owner Tux Class which is a type of React class designed to manage application state, interact with stores, manage route params, and pass props into its ownee components
+//@param ownerClassProps OBJECT: properties that the new ownerClass will possess, should be a standard React.createClass object with properties such as componentWillMount, componentDidMount, etc.  Please see http://facebook.github.io/react/docs/component-specs.html for a full list of React props
```

Comment:
Recommend mentioning the requirement of the render method on the React.createClass object.